controllers/api.js

```diff
@@ -14,6 +14,8 @@ const lob = require('lob')(process.env.LOB_KEY);
 const ig = require('instagram-node').instagram();
 const axios = require('axios');
 const { google } = require('googleapis');
+const refresh = require('passport-oauth2-refresh');
+const User = require('../models/User');
```

Comment:
Can the refresh-related worked be refactored so it is handled within passport.js config, and not in the controller/api.js?  The controller here gets ripped out of the project during code cleanup, and the refresh function seems to be a core functionality that might be needed anytime that we are dealing with an expiring token.  I suspect if it is refactored that way, then there wouldn't be a need for a dependency on passport-oauth2-refresh or ../models/User here.

I haven't thought about this much, but could the refresh potentially be a part of the isAuthorized method in passport.js config?

Reply:
I'll think about it.